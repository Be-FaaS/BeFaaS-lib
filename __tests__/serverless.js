/* eslint-env jest */
const serverless = require('../serverless')
const request = require('supertest')

test('serverless.event', async () => {
  const { googleHandler } = serverless.event(async event => ({ ok: true, event }))
  const response = await request(googleHandler)
    .post('/call')
    .send({ test: 'event' })
  expect(response.status).toBe(200)
  expect(response.body).toMatchSnapshot()
})
