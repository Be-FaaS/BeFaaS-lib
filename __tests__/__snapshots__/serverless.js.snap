// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serverless.event 1`] = `
Object {
  "event": Object {
    "test": "event",
  },
  "ok": true,
}
`;
